utils/cli-args/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+#[derive(Copy, Clone, Debug, clap::ValueEnum, PartialEq, Eq)]
+pub enum UnstableFeature {
+    Enums,
+    ArrayOwnership,
+}
+
+impl std::fmt::Display for UnstableFeature {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        // Note: clap::ValueEnum uses kebab-case by default so
```

Comment:
https://difftastic.wilfred.me.uk/rustdoc/strum/derive.EnumString.html seems to allow a lot of control, could be an alternative to clap?

Reply:
Not sure how it'd be an alternative to clap if it doesn't handle command line arg parsing. Clap would still need a value_parser

Reply:
What I mean is that you can decide how strings are parsed/displayed here, and we can elsewhere, in the actual CLI, have another construct dedicated where we specify the `value_parser` to be based on strum.

Reply:
https://github.com/clap-rs/clap/discussions/4264 Shows a somewhat verbose example